app/points/calculate-points-for-workload.js

```diff
@@ -8,6 +9,7 @@ module.exports = function (workload, caseTypeWeightings) {
   assertObjectType(caseTypeWeightings, CaseTypeWeightings, 'CaseTypeWeightings')
   var totalWorkloadPoints = calculatePointsForTiers(workload.communityTiers, caseTypeWeightings.pointsConfiguration.communityTierPointsConfig, caseTypeWeightings) +
                             calculatePointsForTiers(workload.custodyTiers, caseTypeWeightings.pointsConfiguration.custodyTierPointsConfig, caseTypeWeightings) +
-                            calculatePointsForTiers(workload.licenseTiers, caseTypeWeightings.pointsConfiguration.licenseTierPointsConfig, caseTypeWeightings)
+                            calculatePointsForTiers(workload.licenseTiers, caseTypeWeightings.pointsConfiguration.licenseTierPointsConfig, caseTypeWeightings) +
+                            calculateArmsPoints(workload.armsLicenseCases, workload.armsCommunityCases, caseTypeWeightings.armsLicense, caseTypeWeightings.armsCommunity)
```

Comment:
PAROMs and SDRs are calculated separately from workload points and then added in `calculate-workload-points`. Any reason ARMs has been combined here? Are PAROMs and SDRs types of cases?